### ISSUE TITLE
attempt to index field 'http' (a nil value)

### ISSUE THREAD

**Issue by xcw0754** [dovecot/documentation#444]
From this page: https://doc.dovecot.org/admin_manual/lua/#lua-api
I see this:

```
dovecot.http.client({timeout=milliseconds, max_attempts=number, debug=boolean})
    Create a new http client object that can be used to submit requests to remote servers.
```

And http client code:

```
...
local http_client = dovecot.http.client {
    timeout = 10000;
    max_attempts = 3;
    debug = true;
}
...
```

I got dovecot's log:

```
Apr  8 02:09:13 ubuntua dovecot: auth: Fatal: passdb-lua: initialization failed: lua_pcall(/etc/dovecot/auth.lua) failed: /etc/dovecot/auth.lua:14: attempt to index field 'http' (a nil value)
```

dovecot version: 2.3.18

The question is: Does dovecot contains http client in lua plugin?

**Comment by cmouse** [comment]
Hi!

This feature will be available in 2.3.19. Seems the documentation isn't clear enough about this.

**Comment by xcw0754** [comment]
> Hi!
> 
> This feature will be available in 2.3.19. Seems the documentation isn't clear enough about this.

OK, Thank you
